web/modules/custom/contact_information/contact_information.module

```diff
@@ -112,29 +112,50 @@ function contact_information_entity_auto_label(EntityInterface $entity) {
  * Implements hook_ENTITY_TYPE_access().
  */
 function contact_information_rdf_entity_access(EntityInterface $entity, $operation, AccountInterface $account) {
-  // Access being checked for a contact information.
-  if ($entity->getEntityTypeId() != 'rdf_entity') {
+  if ($entity->bundle() !== 'contact_information') {
     return AccessResult::neutral();
   }
-  if ($entity->bundle() != 'contact_information') {
-    return AccessResult::neutral();
-  }
-  // Only revoke access for anonymous users.
-  if ($account->isAuthenticated()) {
-    return AccessResult::neutral();
+
+  // Inline entity form module is using the 'update' as a name for the edit
+  // operation as it is with all entities apart from nodes that retained the
+  // 'edit' permission for legacy reasons. However, rdf entities also use this
+  // naming convention. Rename 'update' to 'edit' when inline_entity_form checks
```

Comment:
Created issue in RDF Entity to deprecate passing in `edit`: https://github.com/ec-europa/rdf_entity/pull/107

I would suggest to update our code to pass in `update`.

Reply:
We need a solid plan to update to `update` operation instead of `edit`.
The rdf route is still requesting an edit permission. In https://github.com/ec-europa/rdf_entity/pull/107/files we deprecated the 'edit' permission but the route definition is still
```
entity.rdf_entity.edit_form:
  path: '/rdf_entity/{rdf_entity}/edit'
  defaults:
    _entity_form: rdf_entity.edit
    _title: 'Edit Rdf'
  requirements:
    _entity_access: 'rdf_entity.edit'
```
That last part is requesting the `edit` operation.

Reply:
I updated the comment actually. I was wrong to say that this is the case for nodes but the code is still needed until the operations are updated in rdf entity.

Reply:
OK this will start throwing deprecation warnings as soon as the next version of RDF Entity is released, we can then update it.